tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -176,6 +177,35 @@ def test_deployment(self):
         log.info(f"Waiting {wait_time} seconds...")
         time.sleep(wait_time)
 
+        # Create MDS pods for CephFileSystem
+        log.info(f"Enabling Ceph FileSystem")
+        create_oc_resource(
+            'cephfilesystem.yaml', cluster_path, _templating, ENV_DATA
+        )
+        POD = ocp.OCP(kind='Pod', namespace=ENV_DATA['cluster_namespace'])
+        assert POD.wait_for_resource(
+            condition='Running', selector='app=rook-ceph-mds',
+            timeout=600
+        )
+        pods = POD.get(selector='app=rook-ceph-mds')['items']
```

Comment:
line 190 - 194 can be done in wait_for_resource() by just passing 
resource_count=2
and it will do exactly what you did here (:

Reply:
@RazTamir thanks. Will make the changes.

I had a query - shall I keep the rest of the code intact or change them to as per the new chnages (kind class , no jinja2, helper functions, etc.)

Currently, test_ocs_basic_install.py is still using the old methods of resource creations.

Please suggest.

Reply:
no,
Please change everything with the new approach (kind class , no jinja2, helper functions, etc.)
The deployment part which includes test_ocs_basic_install.py will move later

Reply:
@RazTamir sorry i missed your comment and added based on old structure,

Note: The validation part uses the new approach though. Please review once and let me know

Reply:
@RazTamir tried changing the code as per the new recommendation. Please review :)